internal/pkg/handler/create.go

```diff
@@ -13,17 +12,7 @@ type ResourceCreatedHandler struct {
 // Handle processes the newly created resource
 func (r ResourceCreatedHandler) Handle() error {
 	if r.Resource == nil {
-		logrus.Errorf("Error in Handler")
-	} else {
-		logrus.Infof("Detected changes in object %s", r.Resource)
-		// process resource based on its type
-		if _, ok := r.Resource.(*v1.ConfigMap); ok {
-			logrus.Infof("A 'configmap' has been 'Added' but no implementation found to take action")
-		} else if _, ok := r.Resource.(*v1.Secret); ok {
-			logrus.Infof("A 'secret' has been 'Added' but no implementation found to take action")
-		} else {
-			logrus.Warnf("Invalid resource: Resource should be 'Secret' or 'Configmap' but found, %v", r.Resource)
-		}
+		logrus.Errorf("Create Handler received nil object")
```

Comment:
what is create handler @faizanahmad055 ?

Reply:
I have updated it to _Resource creation handler received nil resource_ . Resource creation handler can handle the creation of new resource (i.e. Secret Or Configmap).